docker/debs-builder-docker/build-debs-with-docker.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 #
 # at /path/to/android-cuttlefish/docker, run like this:
-#  ./debs-builder-docker/build-debs-docker.sh
+#  ./debs-builder-docker/build-debs-docker.sh -o /path/to/dir
 #
 
 source "shflags"
 
 DEFINE_boolean verbose "true" "show the stdout/stderr from the sub-tools" "v"
+DEFINE_string out_dir "${PWD}/out" "indicates where to save the built debs" "o"
```

Comment:
A random idea: Couldn't it be based on the path of `build-debs-with-docker.sh`, not `PWD` but `$(realpath $(dirname $0))`?

Not urgent, but I think this change will make our code better, since we won't need `pushd` or `popd` change in the following commit; `Make build-debs-with-docker a standalone tool.`

Reply:
used `${PWD}/out` as default for legacy reason

I think the default should be empty and make the flag mandatory this way the caller always know where the to get the artifacts from, otherwise the caller always need to read into the code to know where the output should be which is not ideal.

Reply:
this is a valid request which could be implemented later on. for this PR though I wanted to keep the same behavior when none of the flags are passed.